src/main/mysql/employeesdb.sql

```diff
@@ -0,0 +1,18 @@
+DROP DATABASE IF EXISTS ea_agile_sprint_FlorianB;
+
+#Change username to yours.
+CREATE DATABASE ea_agile_sprint_FlorianB;
+
+USE ea_agile_sprint_FlorianB;
+
+CREATE TABLE `Role` (
+	`id` smallint NOT NULL PRIMARY KEY AUTO_INCREMENT,
+	`title` varchar(50),
+    `description` varchar(2048),
+    `capability` ENUM('Engineering', 'Platforms', 'Data', 'Artificial Intelligence', 'Cyber Security', 'Workday', 'Experience Design', 'Product', 'Delivery', 'Operations', 'Business Development and Marketing', 'Organisational Strategy and Planning', 'People', 'Commercial and Financial Management', 'Business Services Support'),
```

Comment:
there are going to be future stories which require you to be able to add new capabilities and band levels. using an enum would require a database update to make this work. you should change these to IDs to a capability and a band level table

Reply:
Added the changes you suggested and two new tables for competencies (for user story 5).